src/components/Header/MastheadMenuToggle.tsx

```diff
@@ -24,7 +21,6 @@ const MastheadMenuToggle = ({
         isSidebarOpen={isNavOpen}
         onSidebarToggle={() => {
           setIsNavOpen?.((prev) => !prev);
-          dispatch(onToggle());
```

Comment:
Just for reference, the onToggle was deprecated and no longer did anything. It can be safely removed.